app/models/stacks_file.rb

```diff
@@ -55,7 +55,18 @@ def stacks_rights
   end
 
   delegate :rights, :restricted_by_location?, :stanford_restricted?, :embargoed?,
-           :embargo_release_date, :location, to: :stacks_rights
+           :embargo_release_date, :location, :no_download?, :world_viewable?, to: :stacks_rights
+
+  # types that cannot be viewed when download = 'none'
+  def check_download_types
+    ['https://cocina.sul.stanford.edu/models/document', 'https://cocina.sul.stanford.edu/models/3d']
```

Comment:
Can we put these types in the cocina.rb, similar to how we already have for geo? https://github.com/sul-dlss/stacks/blob/main/app/models/cocina.rb#L56C1-L58

Reply:
We may want to consider making a method like `Cocina#proxied?`, which would return true for media or geo.  Stacks would evaluate the download permissions if they are not proxied.

Reply:
The reason I went with this pdf/3d is because I am unsure about images and those are three types. I was assuming they were what you are calling proxied.

Reply:
I'm calling things proxied if they have a location in the probe service response: https://iiif.io/api/auth/2.0/#probe-service-response

Reply:
Gotcha. Yeah I think this is a good idea assuming I can get clarity on how images work when they are set to download=none. I think they still are viewable.

Reply:
made the field not_proxied? works for this